arm-sql/2014-04-01/swagger/serverDisasterRecoveryConfiguration.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Sql Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration": {
```

Comment:
Apparently this is expected private preview behavior.  From one of their engineers: 
"For AutoDr private preview, you’re seeing the expected behavior:
- By default, servers do not have Disaster Recovery Configuration (AutoDr) objects
- Servers can only have up to one AutoDr object
- Queries made against servers that don’t have an AutoDr object return what you’re seeing
"

Their GA behavior will be be more familiar: a GET against "…/servers/{serverName}/replicationGroups" will return an empty list when there are no replication groups.

I can update the spec to reflect this behavior.

---
In reply to: [83281234](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r83281234) [](ancestors = 83281234)